lang/java/ipc-netty/src/main/java/org/apache/avro/ipc/netty/NettyTransceiver.java

```diff
@@ -343,9 +343,9 @@ private void disconnect(boolean awaitCompletion, boolean cancelPendingRequests,
     try {
       if (channel != null) {
         if (cause != null) {
-          LOG.debug("Disconnecting from " + remoteAddr, cause);
+          LOG.debug("Disconnecting from {}", remoteAddr, cause);
```

Comment:
Is this correct?  It changes from `debug(msg, exception)` to `debug(format, arg1, arg2)`